Tribler/Main/vwxGUI/GuiUtility.py

```diff
@@ -15,18 +15,22 @@
 from Tribler.Core.CacheDB.sqlitecachedb import forceDBThread
 from Tribler.Core.Utilities.search_utils import split_into_keywords
 
+from Tribler.Core.simpledefs import NTFY_STARTUP_TICK, NTFY_CREATE, NTFY_DELETE, NTFY_CLOSE_TICK, NTFY_INSERT
 from Tribler.Main.Utility.GuiDBHandler import startWorker, GUI_PRI_DISPERSY
 from Tribler.Main.Utility.GuiDBTuples import RemoteChannel
 
 from Tribler.Main.vwxGUI import forceWxThread
-from Tribler.Main.vwxGUI.SearchGridManager import TorrentManager, ChannelManager, LibraryManager
 from Tribler.Main.vwxGUI.GuiImageManager import GuiImageManager
 from threading import Lock
+from Tribler.Main.vwxGUI.SearchGridManager import TorrentManager, ChannelManager, LibraryManager
+from Tribler.Main.vwxGUI.gaugesplash import GaugeSplash
 
 
 class GUIUtility(object):
     __single = None
     __single_lock = Lock()
+    START_GAUGE_SPLASH_TICKS = 10
```

Comment:
Maybe these variables (`START_GAUGE_SPLASH_TICKS` and `CLOSE_GAUGE_SPLASH_TICKS`) should be declared above the class? That's what I see in most other Tribler files. The advantage of putting it in the class however is that no other files can access these variables.